addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py

```diff
@@ -23,8 +23,26 @@ def __new__(cls, *args, **kwargs):
         raise RuntimeError("%s should not be instantiated" % cls)
 
     @staticmethod
-    def create(gltf):
-        """Create glTF main method."""
+    def create(gltf, profile=False):
```

Comment:
We can maybe use debug value to enable the profiling?

Reply:
Happy to do that -- what value do I use?

Reply:
Done